sockets/1.2/modules/ROOT/pages/sockets-connector-examples.adoc

```diff
@@ -1,7 +1,7 @@
 = Sockets Examples - Mule 4
 :page-aliases: connectors::sockets/sockets-connector-examples.adoc
 
-Anypoint Connector for Sockets (Sockets Connector) enables you to send or receive transport layer packages through both UDP and TCP sockets. The following examples help you to use the available operations considering the Transport Layer protocol you want to implement in your application.
+Anypoint Connector for Sockets (Sockets Connector) enables you to send or receive transport layer packages through both UDP and TCP sockets. The following examples help you to use the available operations considering the Transport Layer protocol you want to implement in your application, and how to create a custom TPC protocol.
```

Comment:
The following examples show you how to use the available operations while taking into consideration the Transport Layer protocol you want to implement in your application, and how to create a custom TPC protocol.